README.md

```diff
@@ -8,27 +8,31 @@ A collection of XML schema's used by EASY. These schema's are published at
 Maintain sword-v1-packaging
 ---------------------------
 
-The Sword-v1 [service document] (ignore login) refers to the [packaging document]
-located next to the ddm [examples]. The `<article id='mapping'>` in the packaging document
+The Sword-v1 [service document] \(ignore login) refers to the published [packaging document], its source is
+located next to the [examples] directory. The `<article id='mapping'>` in the packaging document
 contains the DDM-EMD mapping. Subsequent articles show help content of the web forms.
 
 The mentioned articles are copy pasted from test results of the legacy components `ddm` and `web-ui`,
 in that order. These results are located at `target/pageDumps/swordPackagingFragmentHelp.html`.
```

Comment:
Isn't `target` one of those folders that always gets excluded by `.gitignore`? Isn't there another link to refer to this html page?